src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -328,11 +328,10 @@ public void ILLink_verify_analysis_warnings_hello_world_app(string targetFramewo
             //since the error experience is not good for a developer
             var warnings = result.StdOut.Split('\n', '\r', ')').Where(line => line.StartsWith("ILLink :"));
             var extraWarnings = warnings.Except(expectedOutput);
-            var missingWarnings = expectedOutput.Except(warnings);
```

Comment:
@danmosemsft - can you also remove the `System.Runtime.Serialization` warnings above, while you are in here?

Or if we don't want to reset CI, I can do it separately.

BTW - thanks for this! This should solve the problem for the future.

Reply:
Oh, maybe you could do that separately so it doesnt reset things.